tests/svtypes/typedef_struct_port.ys

```diff
@@ -0,0 +1 @@
+read_verilog -sv typedef_struct_port.sv
```

Comment:
This test might pass even without the changes in `simplify.cc`. Can we extend it?

Reply:
I extended it to check asserts. Is this enough?

Reply:
I think this test coverage is substantially improved. Thank you!

For things in `test_parser` which the comments say should be signed or unsigned, can we verify this in programmatically in the test? I could imagine checking how the ports sign extend or asserting the equivalence of certain declarations would work. Perhaps it would be possible to inspect the presence of the `signed` attribute on the generated RTLIL wires?

Reply:
I updated tests to assert with expected signed/unsigned value.